applications/GPIO/stepcounter/manifest.yml

```diff
@@ -0,0 +1,13 @@
+sourcecode:
+  type: git
+  location:
+    origin: https://github.com/grugnoymeme/flipperzero-StepCounter-fap.git
+    commit_sha: 8e7d7d59420c36ba730c83ebac12e46004d7352d
```

Comment:
here

Reply:
yes that one is the commit i made to this repo... where to find the correct one?

Reply:
it should point to the commit in your repo, with the version you want to add to app catalog. Check out documentation in this repo for details